lib/evmone/instructions_calls.cpp

```diff
@@ -165,6 +165,12 @@ Result extcall_impl(StackTop stack, int64_t gas_left, ExecutionState& state) noe
     stack.push(EXTCALL_ABORT);  // Assume (hard) failure.
     state.return_data.clear();
 
+    // Address space expansion ready check.
+    if (dst_u256 > (uint256{1} << 160) - 1)
+        return {EVMC_OUT_OF_GAS, gas_left};
```

Comment:
hm, on 2nd thought it seems I should add a new `EVMC_` return code for this. Pls confirm if so @chfast

Reply:
Not strictly needed, but I think the `EVMC_ARGUMENT_OUT_OF_RANGE` is good fit for now.